dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -304,7 +305,14 @@ def diff(self, other_profile: CategoricalColumn, options: dict = None) -> dict:
                     other_profile._categories.items(), key=itemgetter(1), reverse=True
                 )
             )
-
+            if cat_count1.keys() == cat_count2.keys():
+                total_psi = 0.0
```

Comment:
if the categories are equal and of equal count the psi is zero. So if there are no categories (and by extension no counts so no percentages to calculate) I have a couple questions:
1) how did the code get called anyway, if there are no categories the categorical profiler should never be initialized and cant be diffed
2) Even if we get here the `psi` of nothing compared to nothing should be zero,  `psi` is used to calculate change between two datasets, if nothing changed because there is nothing in both profiles, returning 0.0 for `psi` I think as a good thing right?